sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/LogsHelper.cs

```diff
@@ -122,5 +142,23 @@ internal static SeverityLevel GetSeverityLevel(LogLevel logLevel)
                     return SeverityLevel.Verbose;
             }
         }
+
+        private static void ExtractProperties(ref string message, IDictionary<string, string> properties, IReadOnlyCollection<KeyValuePair<string, object>> stateDictionary, bool isFormattedMessage)
+        {
+            foreach (KeyValuePair<string, object> item in stateDictionary)
+            {
+                if (item.Key == "{OriginalFormat}")
+                {
+                    if (!isFormattedMessage)
+                    {
+                        message = item.Value.ToString();
+                    }
+                }
+                else
+                {
+                    properties.Add(item.Key, item.Value.ToString());
```

Comment:
key might be empty by accident or intentional.eg: https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/src/OpenTelemetry/Logs/OpenTelemetryLogger.cs#L93